File: kotlin/v3/src/main/kotlin/com/api/v3/persons/dtos/PersonResponseDto.kt
package com.api.v3.persons.dtos

import java.time.LocalDate

data class PersonResponseDto(
    var fullName: String,
    var ssn: String,
    var birthDate: LocalDate,
    var email: String,
    var address: String,
    var gender: String,
    var phoneNumber: String
)

File: c#/v4/People/DTOs/PersonResponseDto.cs
using System.ComponentModel.DataAnnotations;

namespace v4.People.DTOs;

public record PersonResponseDto(
    [Required] string FullName,
    [Required, StringLength(9)] string Ssn,
    [Required] DateOnly BirthDate,
    [Required] string Email,
    [Required, StringLength(10)] string PhoneNumber,
    [Required] string Address
);